preload/ElmSharp.Wearable.preload

```diff
@@ -0,0 +1,10 @@
+###########################################################################################
+## assembly="Tizen";namespace="Tizen.NET";method="Preload";parameter="Param1,Param2,..." ##
+###########################################################################################
+assembly="ElmSharp.Wearable";namespace="ElmSharp.Wearable.CircleDateTimeSelector"
+assembly="ElmSharp.Wearable";namespace="ElmSharp.Wearable.CircleGenList"
+assembly="ElmSharp.Wearable";namespace="ElmSharp.Wearable.CircleScroller"
+assembly="ElmSharp.Wearable";namespace="ElmSharp.Wearable.CircleSpinner"
+assembly="ElmSharp.Wearable";namespace="ElmSharp.Wearable.ICircleWidget"
+assembly="ElmSharp.Wearable";namespace="ElmSharp.Wearable.IRotaryActionWidget"
```

Comment:
- `ElmSharp.Wearable.CircleSpinner` is not a namespace. It's a type (class) name. `ElmSharp.Wearable` is the namespace.

- What happens if a method name or parameter value is omitted?

- The syntax `[module]![namespace].[type].[member/method]` (or `!` as ` `) is often used by such as the debugger or SOS to represent a fully qualified type/member/method name, although I don't have a formal reference for it right now. For example,
  ```
  ElmSharp.Wearable.dll ElmSharp.Wearable.CircleDateTimeSelector
  ```